examples/src/features/collecting_events.rs

```diff
@@ -3,8 +3,8 @@ use casper_event_standard::Event;
 use odra::casper_event_standard;
 use odra::prelude::*;
 use odra::{
-    module::{Module, ModuleWrapper},
-    Variable
+    module::{Module, SubModule},
```

Comment:
Two things are wrong here.
1) `odra::module::Module` should not be imported. Code generated by macros use it, but it uses full path, so no need to import it. Please remove it from examples, and modules.
2) `SubModule` should be imported same way `Var` and `Mapping` so suggest it is `use odra::SubModule`.

Reply:
@zie1ony Doesn't work without importing Module:
```
error[E0599]: no method named `env` found for reference `&collecting_events::Machine` in the current scope
  --> src/features/collecting_events.rs:49:14
   |
49 |         self.env().emit_event(Info {
   |              ^^^ method not found in `&Machine`
   |
   = help: items from traits can only be used if the trait is in scope
help: some of the expressions' fields have a method of the same name
   |
49 |         self.e1.name.env().emit_event(Info {
   |              ++++++++
49 |         self.e2.name.env().emit_event(Info {
   |              ++++++++
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
   |
2  + use odra::module::Module;
   |
```

Reply:
I changed both:
```
use odra::{Submodule, Module}
```